Mathlib/Topology/SubsetProperties.lean

```diff
@@ -1077,9 +1077,44 @@ instance Quotient.compactSpace {s : Setoid α} [CompactSpace α] : CompactSpace
   Quot.compactSpace
 #align quotient.compact_space Quotient.compactSpace
 
+/-- We say that a topological space is a *weakly locally compact space*,
+if each point of this space admits a compact neighborhood. -/
```

Comment:
Could you mention this definition in the module docstring, as well as in the docstring of `LocallyCompactSpace`?